packages/orchestration/src/client/api/schema/response-chat-message.ts

```diff
@@ -8,7 +8,7 @@ import type { MessageToolCalls } from './message-tool-calls.js';
  * Representation of the 'ResponseChatMessage' schema.
  */
 export type ResponseChatMessage = {
-  role?: 'assistant';
+  role: 'assistant';
```

Comment:
[q] I remember we had a workaround for this, is that fixed as well?

Reply:
No, I will check it. Thanks for pointing this out.